src/test/resources/outputs/redshift/basics/exclude.sql

```diff
@@ -6,21 +6,21 @@ SELECT "t"."flds" AS "flds" FROM "default"."EXCLUDE_T" AS "t";
 SELECT "t"."foo" AS "foo" FROM "default"."EXCLUDE_T" AS "t";
 
 --#[exclude-02]
-SELECT "$__EXCLUDE_ALIAS__"."t"."flds" AS "flds", "$__EXCLUDE_ALIAS__"."t"."foo" AS "foo" FROM (SELECT OBJECT('flds', OBJECT('a', OBJECT('field_x', "t"."flds"."a"."field_x", 'field_y', "t"."flds"."a"."field_y"), 'c', OBJECT('field_x', "t"."flds"."c"."field_x", 'field_y', "t"."flds"."c"."field_y")), 'foo', "t"."foo") AS "t" FROM "default"."EXCLUDE_T" AS "t") AS "$__EXCLUDE_ALIAS__";
+SELECT OBJECT_TRANSFORM("t"."flds" KEEP '"a"', '"c"') AS "flds", "t"."foo" AS "foo" FROM "default"."EXCLUDE_T" AS "t";
```

Comment:
A benefit to the new approach in marking which structs to expand is that the query can be much shorter. Could consider adding this logic to the other targets (i.e. Trino and Spark) if we revisit those transpiled queries.